<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginscreen</class>
 <widget class="QMainWindow" name="loginscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family: century gothic;
font-size: 30px;
background-color:#201f1f;
}
QFrame{
background: #605E5C ;
border-radius:10px;

}
QLabel{
color:white;
font-weight: bold;
}
QToolButton{
background:white;
border-radius:60px;
}
QLineEdit{
background: transparent;
border:none;
color:BLACK;
border-bottom:1px solid black;
font-size:20px;
}
QPushButton{
font-size:18px;
font-weight:bold;
color:white ;
background:#201f1f;
}
QPushButton:hover{
color:white;
background-color:black;
}
#logoLabel{
background:transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="loginForm_frame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>350</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>SURVEILIA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_field">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_field">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>390</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="admin_radioButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>92</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
color:#302F2E;
font-size:16px;
background:transparent;


}</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="user_radioButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>305</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
color:#302F2E;
font-size:16px;
background:transparent;

}</string>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginas_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background: transparent;
border:none;
color:#302F2E;
font-size:18px;
font-weight:500;

}</string>
     </property>
     <property name="text">
      <string>Login as:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>150</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource01.qrc">:/logo/surveilialogo.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="logo_toolButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>loginForm_frame</zorder>
   <zorder>logo_toolButton</zorder>
   <zorder>logo_label</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resource01.qrc"/>
 </resources>
 <connections/>
</ui>
